views/index.erb

```diff
@@ -47,13 +270,43 @@
 
 <!-- Add your Javascript to this script tag -->
 <script language="javascript">
-  console.log("I am Javascript!");
 
   var profile_values = document.getElementsByClassName('profile-value');
   var total_words = 0;
+  var current_words = 0;
+    
+  var about_value = profile_values[0];
 
   // TODO - Count the number of words in the profile.
   // hint - look at what the innerHTML property provides.
+    
+  function WordCount(words) {
+      return words.innerHTML.split(" ").length;
+  }
+    
+  total_words  = WordCount(profile_values[0]);
+  total_words += WordCount(profile_values[1]);
+  total_words += WordCount(profile_values[2]);
+  total_words += WordCount(profile_values[3]);
+  total_words += WordCount(profile_values[4]);
```

Comment:
What if we had 50 sections?... Thinking about extensibility is something we'll address on the course